packages/core/src/base/cacheManagerKeyv.ts

```diff
@@ -34,50 +34,24 @@ const addCacheHeaders = (resHeaders, path, key, hash, time): void => {
 };
 
 /**
- * Calculate a reasonable time to live for the response based on the response headers.
+ * Determines the response to be cached or not.
  *
- * @param response The response to calculate the TTL for
+ * @param response The response to determine ability for caching
  *
- * @returns Time to cache the response in seconds, zero for should not be cached
+ * @returns boolean to cache the response or not
  */
-const getTimeToLiveInSeconds = (response: fetch.Response): number => {
+const isCachable = (response: fetch.Response): boolean => {
```

Comment:
I don't love the name "isCachable" here

Technically it is always cacheable.  we just shouldn't.  What about calling it "shouldCache" or flipping it to "doNotCache"

Reply:
Done.